src/System.Console/tests/WindowAndCursorProps.cs

```diff
@@ -31,19 +31,10 @@ public static void BufferSize_GettingSameAsWindowSize()
     }
 
     [Fact]
-    [SkipOnTargetFramework(TargetFrameworkMonikers.Netcoreapp | TargetFrameworkMonikers.Uap)]
-    public static void WindowWidth_WindowHeight_InvalidSize_net46()
-    {
-        Assert.Throws<IOException>(() => Console.WindowWidth = 0);
-        Assert.Throws<IOException>(() => Console.WindowHeight = 0);
-    }
-
-    [Fact]
-    [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
     public static void WindowWidth_WindowHeight_InvalidSize()
     {
-        Assert.Throws<ArgumentOutOfRangeException>("value", () => Console.WindowWidth = 0);
-        Assert.Throws<ArgumentOutOfRangeException>("value", () => Console.WindowHeight = 0);
+        Assert.Throws<ArgumentOutOfRangeException>("width", () => Console.WindowWidth = 0);
```

Comment:
please run the test against NetFX from corefx. this could be environment running issue as I am seeing the failure was because the OS call failed. so there could be some condition causing that.